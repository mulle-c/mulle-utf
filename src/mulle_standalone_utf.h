//
//  mulle_standalone_utf.h
//  mulle-utf
//
//  Created by Nat! on 07.04.16.
//  Copyright Â© 2016 Mulle kybernetiK. All rights reserved.
//

#ifndef mulle_standalone_utf_h__
#define mulle_standalone_utf_h__

#include <mulle_utf/mulle_utf.h>

#endif 
