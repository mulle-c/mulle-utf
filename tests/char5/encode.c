#include <mulle_standalone_utf/mulle_standalone_utf.h>

#include <stdio.h>
#include <stdlib.h>



int  main()
{
   int  c;
   int  i;
   int  d;

   for( i = 0; i < 255; i++)
   {
      c = mulle_char5_encode( i);
      if( c < 0)
            continue;
      d = mulle_char5_decode( c);
      if( d != i)
      {
         printf( "%d failed\n", i);
         abort();
      }

      if( d)
         printf( "%c", d);
   }
   printf( "\n", d);
}

