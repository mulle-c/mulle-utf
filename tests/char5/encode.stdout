.ACDEINOPST_abcdefghilmnoprstuy
3c1
.u
128398a4
DEINOP
